docs/reference/hooks.rst

```diff
@@ -388,17 +388,56 @@ Hooks for customising the editing interface for pages and snippets.
         js_includes = format_html_join('\n', '<script src="{0}"></script>',
             ((static(filename),) for filename in js_files)
         )
+        # remember to use double '{{' so they are not parsed as template placeholders
         return js_includes + format_html(
             """
             <script>
-                $(function() {
+                $(function() {{
                     $('button').raptorize();
-                });
+                }});
             </script>
             """
         )
 
 
+  Override how the title is set when uploading a document or image.
```

Comment:
I don't think this is the right place in the docs for this - it's not describing the functionality of the `insert_editor_js` hook, but rather one particular JS customisation that you might do through it. Perhaps it should be a new subsection in advanced topics -> images (with a pointer to it from advanced topics -> documents)?

Also, I'm not sure that `insert_editor_js` will work for this, because that only gets inserted on pages that include `wagtailadmin/pages/_editor_js.html` - the multiple image uploader (for example) is not one of those. (Ideally `_editor_js.html` would only be used by the image/snippet/modeladmin edit views, although in practice it ends up being included on a bunch of others through lazy copy-and-paste)

Reply:
Agreed, good call, working on this now.